### PR TITLE
feat: Edit README.md file 

### LINKED ISSUE #4: docs: edit README file
Edit README file for the repo explaining that all the technical documentation and research would be in this repository. Research files must be added under the research folder.

Also mention that for further assistance they can join the following Telegram groups:
Semaphore-Stellar (maintainers of Semaphore Stellar): https://t.me/+-9623JNgGjEyNzI5
Semaphore Community (core developers of the Semaphore protocol): @semaphore_community (search it on Telegram

### PR DESCRIPTION
**Pull request by josephpdf**
Closes #4 

## 🔄 Changes Made
The changes made were:
- Update README.md
- Add a structure for the readme, with project description, references, maintainer and groups

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,20 @@
...
+
+## About Semaphore
+Semaphore is a zero-knowledge protocol designed to provide privacy for applications. It allows users to prove membership in a group without revealing their identity, making it ideal for anonymous voting or signaling.
+- Official Semaphore Project: [Semaphore Protocol](https://semaphore.pse.dev/)
+- Source Code: [Semaphore GitHub Repository](https://github.com/semaphore-protocol/semaphore)
+
+## Community Support
+For questions, you may join these groups:
+- Semaphore-Stellar Maintainers: [Join the Telegram Group](https://t.me/+-9623JNgGjEyNzI5)
+- Semaphore Community (core developers): Search @semaphore_community on Telegram.
```
Comment (evgongora) [inline README.md]: Could you please add that we're not linked to Semaphore Core developers but that the group can be used for direct support.

**Comment by evgongora** [comment]
Gj so far @josephpdf! Just a little change.

File: README.md (hunk 1 above)
Comment (josephpdf) [reply README.md]: I chose to delete that line.

**Comment by evgongora** [comment]
Could you please add it again, something like this?
image

**Comment by evgongora** [comment]
LGTM! Great job @josephpdf

**Pull request merged**